integrations/extensions/docs/elasticsearch-install-and-setup/README.md

```diff
@@ -13,143 +13,10 @@ You can follow the guide [here](../../starter-kits/elasticsearch/README.md) to b
-* `$FILTER` is the variable that contains a list of filters defined in the `Advanced Elasticsearch Settings` if it is available.
-* Learn more about ELSER from [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/semantic-search-elser.html)
-* Learn more about Elasticsearch boolean query and filters from [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-bool-query.html)
-
-
-#### b. Dense Vector (KNN) Search
-For dense vector search, use the following knn query body as an example:
-```json
-{
-  "knn": {
-    "field": "text_embedding.predicted_value",
-    "query_vector_builder": {
-      "text_embedding": {
-        "model_id": "intfloat__multilingual-e5-small",
-        "model_text": "$QUERY"
-      }
-    },
-    "k": 10,
-    "num_candidates": 100,
-    "filter" : "$FILTER"
-  }
-}
-```
-
-Notes:
-* `text_embedding.predicted_value` refers to the field that stores the dense vectors. You may need to update it if you use a different field in your index.
-* `text_embedding` under `query_vector_builder` is the natural language processing task to perform. It has to be `text_embedding` for KNN search.
-* `intfloat__multilingual-e5-small` is the embedding model ID. You may need to update it if you want to use a different embedding model.
-* `$FILTER` is the variable that contains a list of filters defined in the `Advanced Elasticsearch Settings` if it is available.
-* Learn more about knn search from [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/knn-search.html).
-* Learn more about how to set up a text embedding model in Elasticsearch from [here](text_embedding_deploy_and_use.md).
-
-#### c. Using nested queries to search over nested documents with ELSER
-The nested queries are used in the [PDF ingestion guide](how_to_use_web_crawler_in_elasticsearch.md) and [web crawl guide]((how_to_index_pdf_and_office_documents_elasticsearch.md)). Please refer to the guides for more details. 
-
-For using nested queries to search over nested documents, use the following query body as an example:
-```json
-{
-  "query": {
-    "nested": {
-      "path": "passages",
-      "query": {
-        "text_expansion": {
-          "passages.sparse.tokens": {
-            "model_id": ".elser_model_2_linux-x86_64",
-            "model_text": "$QUERY"
-          }
-        }
-      },
-      "inner_hits": {"_source": {"excludes": ["passages.sparse"]}}
-    }
-  },
-  "_source": false
-}
-```
-
-For using filters defined in the `Advanced Elasticsearch Settings` in the query body:
-```json
-{
-  "query": {
-    "nested": {
-      "path": "passages",
-      "query": {
-        "bool": {
-          "should": [
-            {
-              "text_expansion": {
-                "passages.sparse.tokens": {
-                  "model_id": ".elser_model_2_linux-x86_64",
-                  "model_text": "What is an action?"
```

Comment:
"What is an action?" should be "$QUERY", I think.

Reply:
This is a removed block.